sdk/cosmos/tests.yml

```diff
@@ -55,71 +54,6 @@ jobs:
 
       TestResultsFiles: '**/junitreports/TEST-*.xml'
 
-
-  # This template parllelizes tests that could be run on Single_Region_Session
-  # by running them against a different cosmosdb instance
-  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
-    parameters:
-      TestName: Single_Region_Session_2
```

Comment:
you seem to be removing our integration tests. are the tests not working for v3?

Reply:
They are: 
* just clubbed some of different CIs to single CI with both Https and Tcp protocol. 
* removed some of the extra ones which we don't need every single time for every change. 
* We might have to do somewhat similar with master tests as well.